src/java.base/share/classes/java/text/DigitList.java

```diff
@@ -194,6 +194,9 @@ public final long getLong() {
         return Long.parseLong(temp.toString());
     }
 
+    /**
+     * Utility routine to get the value of the digit list.
```

Comment:
new BigDecimal("") will also throw a NumberFormatException so I would add a similar comment for count as done above for getLong/Double()

Reply:
Thanks Lance, will make that change